runtime/onert/backend/xnnpack/ops/ConvolutionLayer.cc

```diff
@@ -119,18 +119,32 @@ void ConvolutionLayer::prepare()
     throw std::runtime_error{"failed to create FP32 Convolution operator"};
   }
   assert(_kernel_op != nullptr);
+  result = true;
+  return result;
```

Comment:
How about using `return true` instead of defining variable result?
```suggestion
  return true;
```